SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -19,13 +19,123 @@ DEFINE_LOG_CATEGORY(LogInterestFactory);
 
 namespace SpatialGDK
 {
+struct FrequencyConstraint
+{
+	float Frequency;
+	SpatialGDK::QueryConstraint Constraint;
+};
+static TArray<FrequencyConstraint> CheckoutConstraints;
+
 // The checkout radius constraint is built once for all actors in CreateCheckoutRadiusConstraint as it is equivalent for all actors.
 // It is built once per net driver initialisation.
 static QueryConstraint ClientCheckoutRadiusConstraint;
 
 // Cache the result type of client Interest queries.
 static TArray<Worker_ComponentId> ClientResultType;
 
+QueryConstraint CreateLegacyNetCullDistanceConstraint(USpatialClassInfoManager* ClassInfoManager)
```

Comment:
nit: would prefer these private functions to be below the public methods of the class